metals/src/main/scala/scala/meta/internal/metals/codeactions/StringActions.scala

```diff
@@ -0,0 +1,103 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.meta.internal.metals.CodeAction
+import scala.meta.internal.metals.Buffers
+import scala.meta.internal.metals.MetalsEnrichments._
+import org.eclipse.{lsp4j => l}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.meta.pc.CancelToken
+import scala.meta.tokens.Token
+import scala.meta.internal.metals.Trees
+
+class StringActions(buffers: Buffers) extends CodeAction {
+
+  override def kind: String = l.CodeActionKind.Refactor
+
+  override def contribute(
+      params: l.CodeActionParams,
+      token: CancelToken
+  )(implicit ec: ExecutionContext): Future[Seq[l.CodeAction]] = {
+
+    val uri = params.getTextDocument.getUri
+    val path = uri.toAbsolutePath
+    val range = params.getRange
+
+    Future
+      .successful {
+        val input = path.toInputFromBuffers(buffers)
+        Trees.defaultDialect(input).tokenize.toOption match {
```

Comment:
We want to use the 2.13 dialect, otherwise the feature will not work in a file with Scala 2.13 specific syntax

Reply:
okay perfect :)